repo/linux/andi-i2c

```diff
@@ -0,0 +1,9 @@
+url: t://git.kernel.org/pub/scm/linux/kernel/git/andi.shyti/linux.git
```

Comment:
thanks, do you refer to https or git protocol here?

Similarly is it ok to name this repo as andi-shyti?

Reply:
Hi, sorry, I have made a copy paste mistake this is git://git.kernel.org/pub/scm/linux/kernel/git/andi.shyti/linux.git

Shall I send a new pull request?

It's OK andi-shyti